src/script/util/ComponentUtil.ts

```diff
@@ -78,30 +78,33 @@ export const useKoSubscribable = <T = any>(observable: ko.Subscribable<T>, defau
   return value;
 };
 
-type ChildValues<T extends string | number | symbol> = Record<T, any>;
-
 type Subscribables<T> = {
   [Key in keyof T]: T[Key] extends ko.Subscribable ? T[Key] : never;
 };
 
+type UnwrappedValues<T, S = Subscribables<T>> = {
+  [Key in keyof S]: Unwrapped<S[Key]>;
+};
+
 export const useKoSubscribableChildren = <C extends keyof Subscribables<P>, P extends Record<C, ko.Subscribable>>(
   parent: P,
   children: C[],
-): ChildValues<C> => {
-  const getInitialState = (root: P): ChildValues<C> =>
+): UnwrappedValues<P> => {
+  const getInitialState = (root: P): UnwrappedValues<P> =>
     children.reduce((acc, child) => {
       acc[child] = root?.[child]?.();
       return acc;
-    }, {} as ChildValues<C>);
+    }, {} as UnwrappedValues<P>);
```

Comment:
Might be better with `.reduce<UnwrappedValues<P>(...`

Reply:
Unfortunately this won't stop TypeScript from complaining about the untyped initial accumulator value.